packages/fuchsia_remote_debug_protocol/lib/src/dart/dart_vm.dart

```diff
@@ -10,7 +10,7 @@ import 'package:web_socket_channel/io.dart';
 
 import '../common/logging.dart';
 
-const Duration _kConnectTimeout = const Duration(seconds: 30);
+const Duration _kConnectTimeout = const Duration(seconds: 9);
```

Comment:
Curious: why 9?

Reply:
If there are unresponsive ports it seems like common practice is to poll three times before giving up.  This would make that the case, instead of polling ten times before giving up.